src/common/classes/vector.h

```diff
@@ -135,7 +157,11 @@ class Vector
 
 protected:
 	FB_SIZE_T count;
-	T data[Capacity];
+	union
+	{
+		T data[Capacity];
+		A align;
+	};
 };
```

Comment:
This doesn't compiles on MSVC12:
error C2621: 'Firebird::Vector<Object,16,char>::data' : illegal union member; type 'Jrd::PageNumber' has a copy constructor
...
\src\jrd\../jrd/Record.h(169) : see reference to class template instantiation 'Firebird::Stack<Jrd::PageNumber,16>' being compiled